### ISSUE TITLE
hyperparameter setting for training Wikidata

### ISSUE THREAD

**Issue by binxuan** [thunlp/OpenKE#174]
Hi,

For training knowledge graph embedding on large scale graph like Wikidata, what is the parameter setting you are using, eg. epochs, nbatches, learning rate? And which toolkit is the one you used for large scale graph?

Thanks

**Comment by THUCSTHanxu13** [comment]
In the experiment of the demo paper, we train TransE on Wikidata by out toolkit (https://github.com/thunlp/Fast-TransX).

**Comment by THUCSTHanxu13** [comment]
The dimension is 50/100, the learning rate is 0.001, the margin is 2.0, and the training epochs is 10000.

**Comment by binxuan** [comment]
Thanks for your response. How about the nbatches? I think it is also an important parameter as it directly determines the batch size.

**Comment by THUCSTHanxu13** [comment]
For the implementation of TransE in (https://github.com/thunlp/Fast-TransX), nbatches is not necessary.

**Comment by binxuan** [comment]
Sorry, I don't quite understand. Why does this implementation of TransE not require nbatches? I just checked the code, the Batch size is Len / nbatches. I also tried to run TransE on Wikidata with nbatches 100 and 1000, which leads to a big difference in terms of training loss.

Thanks
